app/presenters/users/dashboard_presenter.rb

```diff
@@ -13,7 +13,10 @@ def props
         user_name: current_user.name,
         preferred_name: current_user.preferred_name,
         user_title: current_user.full_title,
-        issued_certificates: issued_certificate_details
+        issued_certificates: issued_certificate_details,
+        standing_enabled: standing_enabled?,
+        current_standing_name: current_standing&.name || "",
+        current_standing_color: current_standing&.color || ""
```

Comment:
```suggestion
        standing: standing,
```


```rb
def standing 
    return unless standing_enabled? && current_standing.exisits?
    {
        name: current_standing.name,
        color: current_standing.color
     }
end
```

Reply:
Fixed in this https://github.com/pupilfirst/pupilfirst/pull/1468/commits/53096f45f981cd077f663e829c75b331103d3da5